src/D8Utils.js

```diff
@@ -29,19 +38,21 @@ function formatAMPM(date) {
 }
 
 export const formatTime = (startTime, endTime) => {
-  const formatedStartTime = formatAMPM(
+  let formatedStartTime = standardizeTime(startTime);
+  let formatedEndTime = standardizeTime(endTime);
+  formatedStartTime = formatAMPM(
     new Date(
       new Date(
-        new Date(startTime).toLocaleString("en-US", {
+        new Date(formatedStartTime).toLocaleString("en-US", {
```

Comment:
I could be missing something but why were the nested `New Date(`s used?

Reply:
Thanks for the comment! I added a `new Date` object unintentionally 😃